pallets/file-system/src/tests.rs

```diff
@@ -1371,6 +1371,597 @@ mod revoke_storage_request {
     }
 }
 
+mod msp_accept_storage_request {
+
+    use super::*;
+
+    mod success {
+
+        use super::*;
+
+        #[test]
+        fn msp_accept_storage_request_works() {
```

Comment:
It'd be worth testing more test cases such as "Accepts multiple storage requests from one or different users"